README.md

```diff
@@ -148,9 +154,17 @@ Only the required configuration parameters must be provided when you use these t
 You can also create the configuration yourself where also two convenience functions are provided for generating default configurations for *prod* and *test*,
 `CreateMaskinportenProdConfig` and `CreateMaskinportenTestConfig`. Also here will only the required configuration parameters are needed.
 
-#### Logging
+#### Logging from the Fiks-IO client
 Logging is available by providing the Fiks-IO-Client with a ILoggerFactory. Example of this is provided in the ExampleApplication project.
 
+#### Logging from the RabbitMQ client
+The Fiks-IO-Client uses the official RabbitMQ-Client for .NET. This client logs to system EventLog with the eventsource name "rabbitmq-dotnet-client". 
```

Comment:
Er problemstillingen rundt rabbitmq-dotnet-client vs rabbitmq-client løst nå, slik at alle klienter vil få samme eventsource name uavhengig av kjøremiljø?

Reply:
Nei, og ja? Det viste seg at jeg så på "main" branch, hvor de brukte "rabbitmq-client" som source name. Men for 6.x branchen så bruker "rabbitmq-dotnet-client". Hvis vi skulle senere bumpe til 7.x (som jeg tror de vil ta ut fra main) så må vi se om de har samkjørt dette eller om de endrer source name. Nå er det korrekt med "rabbitmq-dotnet-client"